kivy/uix/textinput.py

```diff
@@ -2583,6 +2585,15 @@ def keyboard_on_textinput(self, window, text):
         if self._selection:
             self.delete_selection()
         self.insert_text(text, False)
+        self._is_textedit=False #准备插入，对束edit状态
+ 
+    def keyboard_on_textedit(self, window, text):
+        #print('textedit',text)
+        if len(text)>0:
+            self._is_textedit=True
```

Comment:
```suggestion
            self._is_textedit = True
```
pep8, spaces around `=`